src/apps/projsync.cpp

```diff
@@ -224,7 +224,7 @@ int main(int argc, char *argv[]) {
     }
 
     if (targetDir.empty()) {
-        targetDir = pj_context_get_user_writable_directory(ctx, true);
+        targetDir = std::string(proj_context_get_user_writable_directory(ctx, true));
```

Comment:
I don't think the explicit cast is needed. You can assign a (non-null) const char* to a std::string implicitly